### PR TITLE
refactor: rename some classes inside addon-doc

### PR DESCRIPTION
**Pull request by splincode**
Part of #7753

### CODE REVIEW

File: projects/addon-doc/utils/is-page-group.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import type {TuiDocPage, TuiDocPageGroup} from '@taiga-ui/addon-doc/types';
+import type {TuiDocPageGroup, TuiDocRoutePage} from '@taiga-ui/addon-doc/types';
 
 export function tuiIsPageGroup(
-    page: TuiDocPage | TuiDocPageGroup,
+    page: TuiDocPageGroup | TuiDocRoutePage,
```
Comment (waterplea) [inline projects/addon-doc/utils/is-page-group.ts:4]: Should we use `Route` suffix in all those types? In `TuiDocPages`, `TuiDocPageGroup`?

File: projects/demo/src/modules/app/home/home.component.ts [hunk 2]
```diff
@@ -12,11 +11,11 @@ import {TuiAccordion, TuiTabsModule} from '@taiga-ui/kit';
...
     imports: [
         TuiTabsModule,
-        TuiDocTabComponent,
+        TuiDocTab,
         NgIf,
-        TuiDocCodeComponent,
+        TuiDocCode,
         TuiLink,
-        RouterLink,
+
```
Comment (waterplea) [inline projects/demo/src/modules/app/home/home.component.ts:18]: `RouterLink` is gone, is that expected? Empty line here

File: projects/demo/src/modules/app/landing/index.ts [hunk 3]
```diff
@@ -22,11 +22,12 @@ import {TuiButton, tuiFadeIn} from '@taiga-ui/core';
     standalone: true,
     imports: [
         NgIf,
-        RouterLink,
+
```
Comment (waterplea) [inline projects/demo/src/modules/app/landing/index.ts:25]: Empty line?

File: projects/demo/src/modules/components/accordion/index.ts [hunk 4]
```diff
@@ -16,14 +15,13 @@ import {TuiAccordionExample5} from './examples/5';
 @Component({
     standalone: true,
     imports: [
-        TuiAddonDoc,
+        TuiDemo,
         TuiAccordion,
         TuiAccordionExample1,
```
Comment (waterplea) [inline projects/demo/src/modules/components/accordion/index.ts:20]: Why do we still use old approach in accordion?

File: projects/demo/src/modules/components/carousel/examples/4/index.ts [hunk 5]
```diff
@@ -18,10 +18,11 @@ import {
         TuiDialogModule,
         TuiCarouselComponent,
         TuiLink,
-        RouterLink,
+
```
Comment (waterplea) [inline projects/demo/src/modules/components/carousel/examples/4/index.ts:21]: Another one.

File: projects/demo/src/modules/components/alert/index.ts [hunk 6]
```diff
@@ -20,7 +19,7 @@ import {AlertExampleWithDataComponent} from './examples/4';
 
 @Component({
     standalone: true,
-    imports: [TuiDemo, TuiLink, TuiButton, TuiTextCodeDirective, RouterLink],
+    imports: [TuiDemo, TuiLink, TuiButton, TuiTextCode],
```
Comment (waterplea) [inline projects/demo/src/modules/components/alert/index.ts:22]: Are you sure router link is not used?
https://github.com/taiga-family/taiga-ui/blob/main/projects/demo/src/modules/components/alert/index.html#L129
Please check all the places where you removed it

**Review by waterplea — APPROVED** [review]
Worried about removed router links and empty lines

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/demo/src/modules/components/alert/index.ts (hunk 6 above)
Comment (splincode) [reply projects/demo/src/modules/components/alert/index.ts:22]: RouterLink included inside TuiDemo

File: projects/demo/src/modules/components/accordion/index.ts (hunk 4 above)
Comment (splincode) [reply projects/demo/src/modules/components/accordion/index.ts:20]: I don't know why, need fix it in next PR

**Pull request merged**